script.js

```diff
@@ -42,4 +110,54 @@ const closing = () => {
   menu.style.display = 'none';
 };
 
-closingbutton.addEventListener('click', closing);
\ No newline at end of file
+closingbutton.addEventListener('click', closing);
+
+document.getElementById('cardsid').innerHTML = data.map((array, index) => `
+<div class="card" id="cardsid_${index}">
+<h2 class="projecttitle">${array.name}</h2>
+<p class="projecttext">${array.description}</p>
+<ul class="buttonsproject">
+    <li class="button"><a class="smoke" href="#">${array.technology[0]}</a></li>
+    <li class="button"><a class="smoke" href="#">${array.technology[1]}</a></li>
+    <li class="button"><a class="smoke" href="#">${array.technology[2]}</a></li>
+</ul>
+<div class="cardbottom">
+    <input id="button_${index}" class="orangebottom hovereffect" type="button" value="See Project" onclick="portfoliopopup(${index})">
+</div>
+</div>
+`).join('');
+
+const portfoliopopup = (value) => {
```

Comment:
- [ ] Kindly ensure your design is responsive, ensure the close button in your mobile version is visible in all popups, I got the result below at 500px , it would also be nice to ensure the buttons position is almost the replica of the Figma design
![image](https://user-images.githubusercontent.com/61408860/180260429-22010250-8d61-44d4-a6d1-1dc230e37b87.png)

 [ ] Kindly ensure your popup has a blurred background, as in the design as shown below:
![image](https://user-images.githubusercontent.com/61408860/180261187-7a07a439-2007-415f-ab91-41b2303090ac.png)

Reply:
> * [ ]  Kindly ensure your design is responsive, ensure the close button in your mobile version is visible in all popups, I got the result below at 500px , it would also be nice to ensure the buttons position is almost the replica of the Figma design
>   ![image](https://user-images.githubusercontent.com/61408860/180260429-22010250-8d61-44d4-a6d1-1dc230e37b87.png)
> 
> [ ] Kindly ensure your popup has a blurred background, as in the design as shown below: ![image](https://user-images.githubusercontent.com/61408860/180261187-7a07a439-2007-415f-ab91-41b2303090ac.png)



Hi **seems like firefox has an error** in the `backdrop-filter:`
This is a screeshot of the documentation of the property using Firefox:
![Screen Shot 2022-07-21 at 14 09 06](https://user-images.githubusercontent.com/62531899/180296880-77e725e8-352a-4925-9573-1722fabcda90.png)
 
This is the same documentation page using Chrome:
![Screen Shot 2022-07-21 at 14 06 42](https://user-images.githubusercontent.com/62531899/180297066-63406bf1-1479-4790-8ea2-9fab5d3ab2da.png)

I tried with the other filter of the properties and none seems to work currently on firefox.

Even i tried in windows and mac versions of Firefox.

Reply:
I found a documentation of it and seems like have to be enable in the local client.

> From version 70: this feature is behind thelayout.css.backdrop-filter.enabled preferences (needs to be set to true) and the gfx.webrender.all preferences (needs to be set to true). To change preferences in Firefox, visit about:config.

Reply:
Changes:

-Add `#blurry {filter: blur(9px)}` (line: 85)
- Add `#popid {max-width: 60vw;}` (line: 119)